src/components/MultiSelect/MultiSelect.test.tsx

```diff
@@ -0,0 +1,675 @@
+      selectedItems={selectedItems}
+      onChange={(e) => handleChange(componentId, e.target.id)}
+      onMixedStateChange={(e) => {
+        handleMixedStateChange(e.target.id);
+      }}
+      onClear={() => setSelectedItems({})}
+      onApply={() => null}
+    />
+  );
+};
+
+describe("MultiSelect Dialog", () => {
+  let selectedTestItems;
+  beforeEach(() => (selectedTestItems = {}));
+  it("should have no axe violations", async () => {
+    const { container } = render(
+      <MultiSelect
+        id="test"
+        label="multiSelect-accessibility"
+        variant="dialog"
+        items={items}
+        selectedItems={selectedTestItems}
+        onChange={() => null}
+        onClear={() => null}
+        onApply={() => null}
+      />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("should initially render with provided id", () => {
+    const { container } = render(
+      <MultiSelect
+        id="some-id"
+        label="MultiSelect Label"
+        variant="dialog"
+        items={items}
+        selectedItems={selectedTestItems}
+        onChange={() => null}
+        onClear={() => null}
+        onApply={() => null}
+      />
+    );
+    expect(container.querySelector("#some-id")).toBeInTheDocument();
+    // @QUESTION: what would be a better way of testing this?
+    // @TODO how to do this?
+  });
+
+  it("should initially render with a given label", () => {
+    render(
+      <MultiSelect
+        id="some-id"
+        label="MultiSelect Test Label"
+        variant="dialog"
+        items={items}
+        selectedItems={selectedTestItems}
+        onChange={() => null}
+        onClear={() => null}
+        onApply={() => null}
+      />
+    );
+
+    expect(
+      screen.getByRole("button", {
+        name: /multiselect test label/i,
+      })
+    ).toBeInTheDocument();
+  });
+
+  it("should initially render with open menu if defaultIsOpen prop is true", async () => {
+    render(
+      <MultiSelect
+        id="some-id"
+        label="MultiSelect Test Label"
+        variant="dialog"
+        items={items}
+        defaultIsOpen={true}
+        selectedItems={selectedTestItems}
+        onChange={() => null}
+        onClear={() => null}
+        onApply={() => null}
+      />
+    );
+
+    expect(screen.getAllByRole("checkbox")).toHaveLength(8);
+  });
+
+  it("should open menu when user clicks menu button", () => {
+    render(
+      <MultiSelect
+        id="some-id"
+        label="MultiSelect Label"
+        variant="dialog"
+        items={items}
+        selectedItems={selectedTestItems}
+        onChange={() => null}
+        onClear={() => null}
+        onApply={() => null}
+      />
+    );
+
+    userEvent.click(
+      screen.getByRole("button", {
+        name: /multiselect label/i,
+      })
+    );
+
+    expect(screen.getByRole("dialog")).toBeInTheDocument();
+    expect(screen.getAllByRole("checkbox")).toHaveLength(8);
+  });
+
+  // Not sure this can be tested
```

Comment:
I believe that snapshot does not capture it either. I tested it by adding the prop to any snapshot test, the test still passes. I think it is because `display: block` is added through the theme (see MultiSelectDialog.tsx/ MultiSelectListbox.tsx where `useMultiStyleConfig` is called with `isBlockElement` passed in).